packages/iov-lisk/src/liskconnection.ts

```diff
@@ -396,6 +397,16 @@ export class LiskConnection implements BcpConnection {
     }
   }
 
+  public async getFeeQuote(): Promise<Fee> {
+    return {
```

Comment:
I think we should wrap that in a transaction type switch (see `function buildMsg(tx: UnsignedTransaction): codecImpl.app.ITx` in @iov/bns) and default to an error, such that we don't return the wrong fee accidentally when new transactions types are added.